model/6getarticles-idcomments.models.js

```diff
@@ -1,18 +1,27 @@
 const db = require("../db/connection");
 
 exports.getAllByArticleId = (article_id) => {
-  const query = `
-    SELECT *
-    FROM comments
-    WHERE article_id = $1
-    ORDER BY created_at DESC
-  `;
-
+  if (!article_id.match(/^\d+$/)) {
+    // ^ If the ID is not a valid number
+    return Promise.reject({
+      status: 400,
+      message: "Bad path! ID must be a number",
```

Comment:
It shouldn't be necessary to handle this error using regex, if a value that isn't a number goes into your query string then PSQL will throw an error for you